main.go

```diff
@@ -54,53 +53,45 @@ func envString(key string, def string) (val string) {
 	}
 	return
 }
+
 func envBool(key string, def bool) (val bool) {
-	v, ok := os.LookupEnv(key)
+	val_, ok := os.LookupEnv(key)
 	if !ok {
 		val = def
-	} else {
-		val, _ = strconv.ParseBool(v)
 	}
+	val, _ = strconv.ParseBool(val_)
 	return
 }
 
 func envInt(key string, def int) (val int) {
-	v, ok := os.LookupEnv(key)
+	val_, ok := os.LookupEnv(key)
 	if !ok {
 		val = def
 		return
 	}
-	i64, _ := strconv.ParseInt(v, 10, 64)
+	i64, _ := strconv.ParseInt(val_, 10, 64)
 	val = int(i64)
 	return
 }
 
 func main() {
 	flag.Parse()
 	// setup structured logging
-	l, cleanup, err := log.Init("github.com/packethost/hegel")
+	l, err := log.Init("github.com/packethost/hegel")
 	logger = l.Package("main")
 	if err != nil {
 		panic(err)
 	}
-	defer cleanup()
-
-	grpcConfig, err := packetgrpc.ConfigFromEnv()
-	if err != nil {
-		logger.Error(err, "Failed to get config")
-		panic(err)
-	}
+	defer l.Close()
 
-	grpcAddr, err := grpcConfig.BindAddress()
+	port, err := strconv.Atoi(env.Get("GRPC_PORT", "42115"))
```

Comment:
use `net.parsePort`

Reply:
ok

Reply:
I am not able to use this function since this is not exported.